### ISSUE TITLE
Entering split-screen mode resets playback position

### ISSUE THREAD

**Issue by gudvinr** [jarnedemeulemeester/findroid#389]
**Describe the bug**
If I use [split-screen mode](https://developer.android.com/guide/topics/large-screens/multi-window-support), when app switches from full-screen mode, playback position resets to where it was when playback started:
* If video was just opened, it resets to 00:00
* If I was previously watching video and I opened it from main window of findroid at XX:YY, when I switch to split-screen it resets to XX:YY

**To Reproduce**
1. Open findroid
2. Start video
3. Open "recent tasks" system UI
4. Choose "Split" action in context menu

**Expected behavior**
App switches to split-screen mode with position at the same spot it were before "recent tasks" is opened.  
If playback was active, it should continue. Otherwise, paused.

**Screenshots**
~~If applicable, add screenshots to help explain your problem.~~

**Device info (please complete the following information):**
 - Device: OnePlus 7 Pro
 - Android version: 12
 - App version: 0.11.1
 - Jellyfin version: 10.8.10

**Additional context**
~~Add any other context about the problem here.~~

**Comment by genericlastname** [comment]
I'm not sure if my issue needs a separate bug because I believe the root cause is the same but this also happens when entering/leaving battery saver mode and when reopening the app after switching to another (sometimes). It seems like whenever the its context changes or it's pushed out of memory by another app Findroid has to reload completely.

I'm on a Pixel 5a running Android 13 and Findroid v0.12.0 (20)

**Comment by gudvinr** [comment]
@jarnedemeulemeester I also noticed same issue when device goes to battery saver mode.  
I suspect that it's not related to battery saver itself but happens due to the fact that system also switches theme to dark (aka night mode), which I believe is covered by `android:configChanges="uiMode"`. But I don't know that for sure, it's just my guess.

I decided to not file another issue for that since it seems to be similar to ones that I already made.
